lease/client.go

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
Use the `// ` style of comments for copyright headers, and please use 2015 as the year. We just updated the fleet codebase, so you could just pull from another file on master.